tensorboard/plugins/hparams/tf_hparams_dashboard/tf-hparams-dashboard.ts

```diff
@@ -20,6 +20,10 @@ import '../tf_hparams_main/tf-hparams-main';
 import * as tf_hparams_backend from '../tf_hparams_backend/tf-hparams-backend';
 import {LegacyElementMixin} from '../../../components/polymer/legacy_element_mixin';
 
+const inColab =
+  new URLSearchParams(window.location.search).get('tensorboardColab') ===
```

Comment:
Could you leave a comment here that this is deliberately a side-effect
at module import time to work around the bug in `AppRouting`, as #4213
did for the scalars dashboard?

https://github.com/tensorflow/tensorboard/blob/5c483ac16a69da89a72c4d8e79b1ef42f435301a/tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.ts#L33-L36

Reply:
(I’ve gone ahead and done this.)